enzyme/index.d.ts

```diff
@@ -541,14 +541,14 @@ export interface MountRendererProps {
  * @param node
  * @param [options]
  */
-export function shallow<P, S>(node: ReactElement<P>, options?: ShallowRendererProps): ShallowWrapper<P, S>;
+export function shallow<P, S>(node: ReactElement<P>, options?: ShallowRendererProps): ShallowWrapper<P, any>;
 
```

Comment:
So the ```S``` parameter is meaningless now because it's not actually used in the type, so it can be removed.